nuxeo-coldstorage-web/ftest/features/coldstorage.feature

```diff
@@ -0,0 +1,87 @@
+    
+  Scenario: User with WriteColdStorage permission can send file to cold storage
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission WriteColdStorage for this document
+    When I browse to the document
+    And I click the Send file to cold storage action button
+    Then I can see the confirmation dialog
+    When I click the confirm button in confirmation dialog
+    Then I can see the file is stored in cold storage
+
+  Scenario: User can Cancel the send file to cold storage action in confirmation dialog
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission WriteColdStorage for this document
+    When I browse to the document
+    And I click the Send file to cold storage action button
+    Then I can see the confirmation dialog
+    When I click the cancel button in confirmation dialog
+    Then I can see the file is not stored in cold storage
+
+  Scenario: User with ReadWrite permission can't send file to cold storage
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission ReadWrite for this document
+    When I browse to the document
+    Then I cannot see the Send file to cold storage button
+
+  Scenario: User can send multiple files to cold storage in Workspace view
+    Given I login as "Jack"
+    And I have the following documents
+      | doctype    | title    | path                | file        |
+      | Workspace  | ws       | /default-domain     |             |
+      | File       | sample1  | /default-domain/ws  | sample.png  |
+      | File       | sample2  | /default-domain/ws  | sample.png  |
+      | File       | sample3  | /default-domain/ws  | sample.png  |
+    And I have the following permissions to the documents
+      | permission        | path                        |
+      | WriteColdStorage  | /default-domain/ws/sample1  |
+      | WriteColdStorage  | /default-domain/ws/sample2  |
+      | WriteColdStorage  | /default-domain/ws/sample3  |
+    When I browse to the document with path "/default-domain/ws"
+    And I select the "sample1" document
+    And I select the "sample2" document
+    And I select the "sample3" document
+    And I move the files to cold storage
+    When I browse to the document with path "/default-domain/ws/sample1"
+    Then I can see the file is stored in cold storage
+    When I browse to the document with path "/default-domain/ws/sample2"
+    Then I can see the file is stored in cold storage
+    When I browse to the document with path "/default-domain/ws/sample3"
+    Then I can see the file is stored in cold storage
+
+  Scenario: User without WriteColdStorage permission to a document cannot see the Send the selected files to cold storage action when said document is selected in Workspace view
```

Comment:
hey Perdo, thx for taking into account the code review.
 nice to have -> can we rename a little bit the scenario name for example I was thinking to something like:

`User without WriteColdStorage permission on all selected documents cannot send them to cold storage`

Reply:
Thanks for the suggestion! I struggled to get a good name for the scenario and also think it ended up being too long 😄